commands/copilot_cli.py

```diff
@@ -9,11 +9,21 @@
 from jsonschema import validate as validate_json
 import yaml
 
-from .utils import camel_case, mkdir, mkfile, SSM_BASE_PATH, setup_templates
+from .utils import camel_case, ensure_cwd_is_repo_root, mkdir, mkfile, SSM_BASE_PATH, setup_templates
 
 
 BASE_DIR = Path(__file__).parent.parent
 
+WAF_ACL_ARN_KEY = "waf-acl-arn"
+
+
+def list_copilot_local_environments():
+    return [path.parent.parts[-1] for path in Path("./copilot/environments/").glob("*/manifest.yml")]
+
+
+def list_copilot_local_services():
+    return [path.parent.parts[-1] for path in Path("./copilot/").glob("*/manifest.yml")]
```

Comment:
Would this include things like `environments` and `pipelines`?

Reply:
Ah, maybe not, I think this would return every directory in `copilot` which has a direct descendant called `manifest.yml`?

Reply:
@WillGibson Yes, that is correct. Those directories that contain the `manifest.yml` file which are returned are the Copilot services.